nevergrad/functions/cycling/MensTeamPursuit.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+from .TeamPursuit import TeamPursuit
+from .Cyclist import Cyclist
+from .SimulationResult import SimulationResult
+import sys
+import math
+
+class MensTeamPursuit(TeamPursuit):
+
+	TEAM_SIZE = 4
+	RACE_DISTANCE = 4000
+	LAP_DISTANCE = 250
+	RACE_SEGMENTS = int((2 * (RACE_DISTANCE / LAP_DISTANCE)) - 1)
+	MAXIMUM_TRANSITIONS = RACE_SEGMENTS - 1
+
+	def __init__(self):
+		super().updateAirDensity()
```

Comment:
A couple of things here:
- you should always call the init method of the parent class
- no need to call super().updateAirDensity(), you can directly use self, unless you have overriden it but wants to explicitely call the parent's one
```suggestion
		super().__init__()
		self.updateAirDensity()
```